api/external/applications/applications.proto

```diff
@@ -108,6 +108,27 @@ service ApplicationsService {
     option (chef.automate.api.iam.policy).action = "applications:serviceGroups:delete";
   };
 
+  rpc GetDeleteDisconnectedServicesConfig(GetDeleteDisconnectedServicesConfigReq) returns (PeriodicJobConfig) {
+    option (google.api.http).get = "/beta/retention/service_groups/delete_disconnected_services/config";
+    // TODO: need new resources/policies for this
+    option (chef.automate.api.policy).resource = "service_groups";
+    option (chef.automate.api.policy).action = "list";
+    option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
+    option (chef.automate.api.iam.policy).action = "applications:serviceGroups:list";
+  };
+
+  rpc UpdateDeleteDisconnectedServicesConfig(PeriodicJobConfig) returns (UpdateDeleteDisconnectedServicesConfigRes) {
+    option (google.api.http) = {
+      post: "/beta/retention/service_groups/delete_disconnected_services/config"
+      body: "*"
+    };
+    // TODO: need new resources/policies for this
+    option (chef.automate.api.policy).resource = "service_groups";
+    option (chef.automate.api.policy).action = "delete";
+    option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
+    option (chef.automate.api.iam.policy).action = "applications:serviceGroups:delete";
```

Comment:
Assuming I am parsing correctly--this is an *update* operation--then L127 and L129 should likely say `update` instead of `delete`. I realize you just copy/pasted this from L90-L109, so my comment applies to those as well--should be changed there, too.

Reply:
Thanks for reviewing! I think I'm ok to let these be horribly wrong in one additional way for now. We have #1371 at the top of the queue to fix the mess :)